docs/_source/guides/llms/practical_guides/fine_tune.md

```diff
@@ -39,13 +39,14 @@ trainer.predict("This is awesome!")
 
 We plan on adding more support for other tasks and frameworks so feel free to reach out on our Slack or GitHub to help us prioritize each task.
 
-| Task/Framework                   | TRL  | OpenAI | SetFit | spaCy | Transformers | PEFT |
+| Task/Framework                  | TRL  | OpenAI | SetFit | spaCy | Transformers | PEFT |
 |:--------------------------------|:-----|:-------|:-------|:------|:-------------|:-----|
-| Text Classification             |      | ✔️      |  ✔️      | ✔️     | ✔️            | ✔️    |
-| Supervised Fine-tuning          | ✔️    |        |          |       |              |      |
-| Reward Modeling                 | ✔️    |        |          |       |              |      |
-| Proximal Policy Optimization    | ✔️    |        |          |       |              |      |
-| Direct Preference Optimization  | ✔️    |        |          |       |              |      |
+| Text Classification             |      |        |  ✔️     | ✔️     | ✔️            | ✔️    |
```

Comment:
Is `Text Classification` not supported for OpenAI now?

Reply:
Not for the `FeedbackTask`. I did not want to spend time on it due to the legacy fine-tuning methodology.